app/Actions/DataFixes/CleanErrantStreamMessages.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Actions\DataFixes;
+
+use App\DataExchange\Models\StreamMessage;
+
+class CleanErrantStreamMessages
+{
+    public function handle()
+    {
+        // Delete errant member_retired for Aurora Pujol on group Peroxisomal Disorders VCEP
+        StreamMessage::find(4034)->delete();
```

Comment:
We will have to have a better way of identifying these records for deletion beside the id... because of the prior migration/reindexing, these messages will have moved to a different id (at least they appear to have done so in my test environment). Maybe they could at least be identified by timestamp?

Reply:
What if we run this in a migration before reindexing?  In fact, we could run it before we do anything else to be sure.